src/R2Architecture.cpp

```diff
@@ -31,6 +31,8 @@ static const std::map<std::string, std::string> arch_map = {
 		{ "ppc64", "PowerPC" } ,
 		{ "sparc32", "sparc" } ,
 		{ "sparc64", "sparc" } ,
+		{ "sh32", "SuperH4" } ,
+		{ "sh32", "SuperH" } ,
```

Comment:
This cannot work, `std::map` can only have one value per key. For now I would say just use one of these and the other will need to be manually selected.